src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Circle extends Shape {
+    private String color;

```

Comment:
```suggestion
public class Circle extends Figure {
```
filed "color" should be in Figure class